package.json

```diff
@@ -8,7 +8,7 @@
     "build": "babel lib -d dist",
     "lint": "eslint lib/",
     "serve": "node dist/index.js",
-    "test": "mocha lib/",
+    "test": "mocha lib/ --compilers js:babel-register --require babel-polyfill",
```

Comment:
You don't need to specify the flags this way anymore. This is superseded by https://github.com/redbadger/badger-brain/blob/master/test/mocha.opts